Cue/app/tabs/library/LibraryHome.js

```diff
@@ -76,10 +80,27 @@ class LibraryHome extends React.Component {
   componentWillReceiveProps(newProps: Props) {
     this.setState({
       refreshing: false
-    })
+    })
+
+    if (newProps.inaccessibleDecks && newProps.inaccessibleDecks.length > 0) {
+      newProps.inaccessibleDecks.forEach(deck => {
+        Alert.alert(
+          (Platform.OS === 'android' ?
+            'The deck “' + deck.name + '” is no longer available from the original owner' :
+            'The Deck “' + deck.name + '” Is No Longer Available From The Original Owner'),

```

Comment:
Up to you, as long as in most cases we don't run into text truncation in the most common use cases.

Also: `from the` is not capitalized in Title Case.